src/app/external-cluster-wizard/steps/external-cluster/provider/gke/template.html

```diff
@@ -63,20 +66,72 @@
       </mat-error>
     </mat-form-field>
 
-    <mat-form-field>
-      <mat-label>Initial Node Count
-      </mat-label>
-      <input matInput
-             type="number"
-             [formControlName]="Controls.NodeCount" />
-      <mat-hint>Set the Initial Node Count.</mat-hint>
-      <mat-error *ngIf="form.get(Controls.NodeCount).hasError('required')">
-        Initial Node Count is <strong>required</strong>.
-      </mat-error>
-    </mat-form-field>
+    <km-autocomplete *ngIf="isDialogView()"
+                     label="Disk Types"
+                     [options]="diskTypes"
+                     [formControlName]="Controls.DiskTypes">
+      <mg-container hint>
+        Set Disk Type
+      </mg-container>
+    </km-autocomplete>
+
+    <km-autocomplete *ngIf="isDialogView()"
+                     label="Machine Sizes"
+                     [options]="machineSizes"
+                     [formControlName]="Controls.MachineSizes">
+      <mg-container hint>
+        Set Machine Size
+      </mg-container>
+    </km-autocomplete>
+
+    <km-autocomplete *ngIf="isDialogView()"
+                     label="Images"
+                     [options]="clusterImages"
+                     [formControlName]="Controls.ClusterImages">
+      <ng-container hint>
+        Set Image
+      </ng-container>
+    </km-autocomplete>
+
+    <km-number-stepper hint="Set Number of Replicas"
+                       mode="all"
+                       [label]="isDialogView() ? 'Number of Replicas' : 'Initial Node Count'"
+                       [max]="form.get(Controls.EnableAutoScaling).value ? form.get(Controls.MaxCount).value : AUTOSCALING_MAX_VALUE"
+                       [min]="form.get(Controls.EnableAutoScaling).value ? form.get(Controls.MinCount).value : AUTOSCALING_MIN_VALUE"
+                       [required]="true"
+                       [formControlName]="Controls.NodeCount">
+    </km-number-stepper>
+
+    <mat-checkbox [formControlName]="Controls.EnableAutoScaling"
+                  (change)="onEnableAutoScalingChange($event)">
+      Enable Auto Scaling
+      <i class="km-icon-info km-pointer"
+         matTooltip="Indicates whether to enable auto scaling.This option is recommended so that cluster sized correctly for the current running workloads."></i>
+    </mat-checkbox>
+
+    <ng-container *ngIf="form.get(Controls.EnableAutoScaling).value">
```

Comment:
Check **AKS** Cluster wizard

OR do we need this Create wizard

![move right side](https://user-images.githubusercontent.com/17727069/181449566-950a863b-b571-4fb1-a143-0315c4a03b21.png)

Reply:
no we don't need it here